CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -171,9 +171,13 @@ CF_PRIVATE CFIndex __CFActiveProcessorCount();
 #define __builtin_unreachable() do { } while (0)
 #endif
 
-#if defined(__i386__) || defined(__x86_64__)
-    #if defined(__GNUC__)
-        #define HALT do {asm __volatile__("int3"); kill(getpid(), 9); __builtin_unreachable(); } while (0)
+#if defined(__i386__) || defined(__x86_64__) || defined(__s390x__)
```

Comment:
It looks like there is always a fall back when the architecture is not i386/x86_64 so we can continue on s390x.  Need to take a look and see if there's work to be done for s390x in those places.  In the mean time, can we please fix this so our s390x build can continue?  Thanks.